File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/YouNeedToStayConfident.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class YouNeedToStayConfident : StoryChoice
	{
		internal YouNeedToStayConfident() : base(StoryChoice.GetLocalizedString("ch5YouNeedToStayConfident"), 5026)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredLiteratureStory = LiteratureStoryAnswer.StayConfident;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/YouNeedToStayConfident.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.LiteratureStoryAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class YouNeedToStayConfident extends StoryChoice {
    public YouNeedToStayConfident() {
        super(R.string.ch5YouNeedToStayConfident, 5026);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredLiteratureStory(LiteratureStoryAnswer.StayConfident);
        super.select();
    }
}
